sockeye/training.py

```diff
@@ -587,6 +588,18 @@ def _load_trainer_states(self, fname):
         self.trainer.load_states(fname)
         logger.info('Loaded optimizer states from "%s"', fname)
 
+    def _save_lr_scheduler(self, fname):
+        with open(fname, "wb") as fp:
+            pickle.dump(self.trainer.optimizer.lr_scheduler, fp)
+        logger.info('Saved lr_scheduler to "%s"', fname)
+        logger.info('Saved lr_scheduler %s', self.trainer.optimizer.lr_scheduler.__repr__())
+
+    def _load_lr_scheduler(self, fname):
```

Comment:
can we make this method backwards compatible? Right now, it would be failing when trying to continue training an existing (2.3.5 and lower) model. You could check in the method for existence of the file under `fname` and only load if it exists.

Reply:
ok